packages/bun-debug-adapter-protocol/src/debugger/adapter.ts

```diff
@@ -489,7 +490,14 @@ export class DebugAdapter extends EventEmitter<DebugAdapterEventMap> implements
           ...env,
         };
 
-    const url = `ws+unix://${randomUnixPath()}`;
+    let url: string;
+
+    if (getOSType() === "Windows_NT") {
+      url = `ws+unix:///${randomUnixPath()}`.replaceAll("\\", "/");
```

Comment:
Why not use standard `ws://` for windows instead of `ws+unix`? 
Then update [signal.ts](https://github.com/oven-sh/bun/blob/214b3ccca052203211e18f9d55796549c135edf3/packages/bun-debug-adapter-protocol/src/debugger/signal.ts) to match this change?

I am not familiar with all the project code so this is may not be possible but thought to mention.

Reply:
I believe it's possible! I've started working towards this. There are several more changes necessary (including in the Bun runtime) to support Windows. Someone can ping me if they think it's taking me too long **and** they want to pick up the effort from me; not sure if it's going to be days or weeks as it's not an immediate need for me anymore.

Reply:
Great work on starting this @mikeseese! I have mentioned your reply in the [active issue](https://github.com/oven-sh/bun/issues/8851).  I am in a similar situation as I am tied up with Company related work. I may contribute to/help with this when I have more time

Reply:
> Why not use standard `ws://` for windows instead of `ws+unix`? Then update [signal.ts](https://github.com/oven-sh/bun/blob/214b3ccca052203211e18f9d55796549c135edf3/packages/bun-debug-adapter-protocol/src/debugger/signal.ts) to match this change?
> 
> I am not familiar with all the project code so this is may not be possible but thought to mention.

Tried `ws:`, but seems that it still doesn't work. See [#8851](https://github.com/oven-sh/bun/issues/8851#issuecomment-2264959847)